modules/ROOT/pages/_partials/cloudhub-apps.adoc

```diff
@@ -151,8 +145,7 @@ This command accepts only the default flag `--help`.
 > runtime-mgr:cloudhub-application:deploy  [flags] <name> <zipfile>
 ----
 
-This command deploys the Mule deployable archive ZIP file that you specify in `<zipfile>` using the name you set in `<name>`. +
-You can start typing your application's name and press `tab` for Anypoint Platform CLI to autocomplete it, or you can double tap `tab` for a full list of all the values you can pass. +
+This command deploys the Mule deployable archive ZIP file that you specify in `<zipfile>` using the name you set in `<name>`. 
```

Comment:
I would consider reordering this if it doesn't mess with the parallelism of the rest of the doc. So, say "Using the name you set in '<name>', this command deploys..."